tests/test_dataset_mask.py

```diff
@@ -53,7 +53,7 @@ def tiffs(tmpdir):
     _profile = {
         'affine': Affine(5.0, 0.0, 0.0, 0.0, -5.0, 0.0),
         'transform': Affine(5.0, 0.0, 0.0, 0.0, -5.0, 0.0),
-        'crs': {'init': 'epsg:4326'},
+        'crs': CRS(),
```

Comment:
@jameshiebert please make this explicit rather than implicit:

`'crs': CRS({'init': 'epsg:4326'})`